### PR TITLE
[R4ML-155], [R4ML-159] partial arguments and conf.int -> alpha

### PR DESCRIPTION
**Pull request by nilmeier**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by bdwyer2** [comment]
@nilmeier why don't you close PR #5 since we accidentally duplicated the commits here?

**Review by bdwyer2 — APPROVED** [review]

**Comment by aloknsingh** [comment]
@nilmeier @bdwyer2 can we confirm that these are the only def usage because seems like in the orig jira there was a bigger list

**Comment by bdwyer2** [comment]
@aloknsingh your right. Here are the places that still need to be fixed:

```R4ML/R/ml.model.als.R:68:setMethod("r4ml.model.validateTrainingParameters", signature="r4ml.als", def =
R4ML/R/ml.model.als.R:115:setMethod("r4ml.model.buildTrainingArgs", signature="r4ml.als", def =
R4ML/R/ml.model.als.R:155:setMethod("r4ml.model.postTraining", signature = "r4ml.als", def =
R4ML/R/ml.model.base.R:46:setGeneric("r4ml.model.validateTrainingParameters", def =
R4ML/R/ml.model.base.R:58:setGeneric("r4ml.model.buildTrainingArgs", def =
R4ML/R/ml.model.base.R:69:setGeneric("r4ml.model.postTraining", def =
R4ML/R/ml.model.base.R:84:setGeneric("r4ml.model.import", def =
R4ML/R/ml.model.base.R:238:setGeneric("r4ml.model.getResponseVariable", def =
R4ML/R/ml.model.glm.R:276:  def =
R4ML/R/ml.model.glm.R:327:  def =
R4ML/R/ml.model.glm.R:357:  def =
R4ML/R/ml.model.lm.R:122:setMethod("r4ml.model.validateTrainingParameters", signature="r4ml.lm", def =
R4ML/R/ml.model.lm.R:156:setMethod("r4ml.model.buildTrainingArgs", signature="r4ml.lm", def =
R4ML/R/ml.model.lm.R:203:setMethod("r4ml.model.postTraining", signature="r4ml.lm", def =
R4ML/R/ml.model.lm.R:251:setMethod("coef", signature="r4ml.lm", def =
R4ML/R/ml.model.lm.R:265:setMethod("stats", signature="r4ml.lm", def =
R4ML/R/ml.model.mlogit.R:108:setMethod("r4ml.model.validateTrainingParameters", signature = "r4ml.mlogit", def =
R4ML/R/ml.model.mlogit.R:142:setMethod("r4ml.model.buildTrainingArgs", signature = "r4ml.mlogit", def =
R4ML/R/ml.model.mlogit.R:180:setMethod("r4ml.model.postTraining", signature = "r4ml.mlogit", def =
R4ML/R/ml.model.mlogit.R:224:setMethod("coef", signature="r4ml.mlogit", def =
R4ML/R/ml.model.pca.R:116:  def =
R4ML/R/ml.model.pca.R:151:  def =
R4ML/R/ml.model.pca.R:203:  def =
```

File: R4ML/R/ml.model.base.R [hunk 1]
```diff
@@ -180,7 +180,7 @@ setMethod("initialize", "r4ml.model",
     .Object@yColname <-
       if (isSupervised == TRUE) {
          r4ml.debug(logSource, "Getting response variable...")
-         r4ml.model.getResponseVariable(formula=formula, data=data, model=.Object)
+         r4ml.model.getResponseVariable(formul = formula, data = data, model = .Object)
```
Comment (aloknsingh) [inline R4ML/R/ml.model.base.R:183]: name == 'formula' and NOT 'formul'

Comment (aloknsingh) [reply R4ML/R/ml.model.base.R:183]: i will fix it for now

**Review by aloknsingh — APPROVED** [review]

**Pull request closed without merging**